File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.java
class Solution {
    public int longestConsecutive(int[] nums) {
        int n=nums.length;

        Map<Integer,Integer> mp=new HashMap<>();

        for(int i=0;i<n;i++)
        {
            mp.put(nums[i],i);
        }

        int result=0;

        for(int i=0;i<n;i++)
        {
            if(mp.containsKey(nums[i]-1))   continue;
            int count=1;
            int num=nums[i];
            while(mp.containsKey(num+1))
            {
                count++;
                num++;
            }
            result=Math.max(count,result);
        }
        return result;
    }
}

File: Data Structures & Algorithms/majority-element/submission-2.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        
        count = 0

        element = nums[0]

        for i in range(len(nums)):

            if element == nums[i]:
                count += 1
            
            else:
                count -= 1
            if count == 0:
                element = nums[i]
                count = 1
        
        return element